plugins/node/opentelemetry-plugin-mongodb/src/mongodb.ts

```diff
@@ -192,13 +192,18 @@ export class MongoDBPlugin extends BasePlugin<typeof mongodb> {
     });
 
     if (command === undefined) return;
-    const query = Object.keys(command.query ?? command.q ?? command).reduce(
-      (obj, key) => {
-        obj[key] = '?';
-        return obj;
-      },
-      {} as { [key: string]: string }
-    );
+
+    // capture parameters within the query as well if enhancedDatabaseReporting is enabled.
+    const query = this._config?.enhancedDatabaseReporting
+      ? command.query ?? command.q ?? command
```

Comment:
Nit: line 198 and 199 has the same maybe consider this to put into some variable and then reuse ?
`const commandObj = command.query ?? command.q ?? command;`

Reply:
done